src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -51,32 +53,50 @@ const NavHeader = (props) => {
     }
   };
 
-  const onDomainLayout = (event) => {
-    const {height} = event.nativeEvent.layout;
-    setDomainHeight(height);
-  };
+  const onDomainLayout = React.useCallback(
+    (event) => {
+      const {height} = event.nativeEvent.layout;
+      setDomainHeight(Math.round(height));
+    },
+    [setDomainHeight]
+  );
 
   const backScreen = () => {
     navigation.goBack();
   };
 
   const getBottomPostition = () => {
-    let bottom = 0;
+    let containerHeight = 0;
     if (isHeaderHide) {
-      bottom = dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT2;
+      containerHeight = dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT2;
     } else {
-      bottom = hideSeeMember
-        ? dimen.size.TOPIC_FEED_HEADER_HEIGHT
-        : dimen.size.TOPIC_FEED_HEADER_HEIGHT + normalize(4);
+      containerHeight =
+        hideSeeMember || !isFollow
+          ? dimen.size.TOPIC_FEED_HEADER_HEIGHT
+          : dimen.size.TOPIC_FEED_HEADER_HEIGHT + normalize(4);
     }
-    return (bottom - domainHeight) / 2;
+    return Math.round((containerHeight - domainHeight) / 2);
+  };
+
+  const heightWithCoverImage = () => {
+    if (coverPath) {
+      return {height: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER};
+    }
+    return null;
   };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `heightWithCoverImage` function is a new addition that adjusts the height of the component based on whether a cover image is present. This is a good approach to handle dynamic styling based on state.
```diff
+ const heightWithCoverImage = () => {
+   if (coverPath) {
+     return {height: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER};
+   }
+   return null;
+ };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->